akka-http-core/src/main/scala/akka/http/impl/util/package.scala

```diff
@@ -127,6 +127,66 @@ package object util {
 
 package util {
 
+  import akka.http.scaladsl.model.{ ContentType, HttpEntity }
```

Comment:
I simply don't like hiding things in package.scala, why not use ordinary class files?